wolfcrypt/src/aes.c

```diff
@@ -11371,7 +11386,7 @@ int wc_AesXtsDecrypt(XtsAes* xaes, byte* out, const byte* in, word32 sz,
         }
 
         /* stealing operation of XTS to handle left overs */
-        if (sz > 0) {
+        if (sz >= AES_BLOCK_SIZE) {
```

Comment:
Can you double check this change? `sz` is decremented above and appears like it could be / should be `sz > 0`. @JacobBarthelmeh , thoughts?